driverconfig/config.yaml

```diff
@@ -142,8 +143,16 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: powerstore
     configVersions:
-      - configVersion: v2.4.0
+      - configVersion: v2.5.0
         useDefaults: true
+        supportedVersions:
+          - version: v121
+          - version: v122
+          - version: v123
+          - version: v124
+          - version: v125
+      - configVersion: v2.4.0
+        useDefaults: false
```

Comment:
Please remove all entries related to v2.2.0 as operator supports only n-2 versions of the driver

Reply:
Done.